elixir/apps/web/lib/web/live/settings/account.ex

```diff
@@ -71,24 +72,61 @@ defmodule Web.Settings.Account do
       <:title>
         Danger zone
       </:title>
+      <:action>
+        <.button_with_confirmation
+          id="delete_account"
+          style="danger"
+          icon="hero-trash-solid"
+          on_confirm="delete_account"
+          disabled={@delete_requested}
+        >
+          <:dialog_title>Confirm Account Deletion</:dialog_title>
+          <:dialog_content>
+            This Account will be scheduled for complete deletion.<br /><br />
+            Are you sure you want to delete your Account?
```

Comment:
```suggestion
            The account <strong>{@account.slug}</strong> will be scheduled for permanent deletion.<br /><br />
            Are you sure you want to delete your account?
```

Might be a good idea to remind them of the slug right there, in case they are managing multiple accounts for some legitimate reason.

Reply:
Good suggestion 👍